crud/insert_many.lua

```diff
@@ -98,7 +113,11 @@ local function insert_many_on_storage(space_name, tuples, opts)
                     return nil, errs, replica_schema_version
                 end
 
+                local unref_ok, bucket_unref_err = unref(bucket_ids)
                 box.commit()
+                if not unref_ok then
+                    return nil, bucket_unref_err
+                end
```

Comment:
Nit: in 3 places, where unref stage is repeated several times. Can we do `goto error` here? Smth like:

```lua
   if smth then
       goto error
   end
   <...>
error:
    <unref logic>
    return inserted_tuples, errs, replica_schema_version
```

In order not to duplicate the code